iuf

```diff
@@ -89,7 +89,7 @@ def validate_stages(config):
     # the variables are resolved in that stage.  Further error-checking
     # related to this is done in SiteConfig.
     if any(stage in local_stages for stage in ["update-vcs-config",
-        "update-cfs-config", "management-nodes-rollout"]):
```

Comment:
i haven't looked, but i recall we try to update site vars between every stage.  i wonder if we should (if we have not already) have that site vars update only run if the bootprep files have been provided